src/main/java/com/sahokia/kos/jdbchomework/JDBCHomeworkApplication.java

```diff
@@ -0,0 +1,27 @@
+package com.sahokia.kos.jdbchomework;
+
+import com.sahokia.kos.jdbchomework.dao.TicketServiceDAO;
+import com.sahokia.kos.jdbchomework.model.Ticket;
+import com.sahokia.kos.jdbchomework.model.TicketType;
+import com.sahokia.kos.jdbchomework.model.User;
+
+import java.sql.Date;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+
+public class JDBCHomeworkApplication {
+    public static void main(String[] args) {
+        TicketServiceDAO ticketServiceDAO = new TicketServiceDAO();
+        ticketServiceDAO.saveUser("Vanya", Date.valueOf(LocalDate.now()));
+        ticketServiceDAO.saveTicket(7, TicketType.DAY, Date.valueOf(LocalDate.now()));
+        ticketServiceDAO.saveTicket(7, TicketType.YEAR, Date.valueOf(LocalDate.now()));
+        ticketServiceDAO.saveUser("Tomas", Date.valueOf(LocalDate.now()));
+        List<Ticket> vanyaTickets = ticketServiceDAO.getTicketsByUserId(7);
```

Comment:
Why vanya?))

Reply:
Have changed the name of the variable to just "user", because we don't know in common, what the exact user would be fetched by id "7"